www/js/control/general-settings.js

```diff
@@ -19,7 +20,8 @@ angular.module('emission.main.control',['emission.services',
                                         'emission.survey.enketo.demographics',
                                         'emission.plugin.logger',
                                         'emission.config.dynamic',
-                                        QrCode.module])
+                                        QrCode.module,
+                                        ControlDataTable.module])
```

Comment:
This is where the modulepath is being used.
`ControlDataTable.module` is `'emission.main.control.dataTable'`